docs/DeveloperGuide.md

```diff
@@ -536,6 +536,13 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Listing all students
```

Comment:
Just want to check, what is the intended output when i do list timetable, then find s/[StudentID]? Will the timetable remain visible after i output another command?

Reply:
Currently the timetable will only be visible under command `list timetable`